sdk/cognitiveservices/azanomalydetector/version.go

```diff
@@ -0,0 +1,11 @@
+//go:build go1.18
+// +build go1.18
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package azanomalydetector
+
+const (
+	version = "v3.0.0-beta6"
```

Comment:
Initial version would be `v0.1.0`

Reply:
I made this consistent with the version used in all the other languages, but I'm not opposed to using a go-specific version.

Reply:
Having a major version greater than `1` activates [semantic import versioning](https://github.com/golang/go/wiki/Modules#semantic-import-versioning) and would require us to release the module as `v3`.
```
module github.com/Azure/azure-sdk-for-go/sdk/cognitiveservices/azanomalydetector/v3
```
I don't think we want to go down this path just to align with the other languages (that's been our position so far anyways).